feat: export wallet

potato-khannah-shaltiel opened the pull request:
before: export functionality non-existent 

now: added basic export functionality so the user can export their wallet. once the user presses export wallet, a window opens prompting the user on where to save the wallet to. once they indicate their preferred location, and the export is successful, a green message shows up saying successful export and the file is saved. otherwise, a red message with export failed shows up. the wallet is exported as a JSON in the following format: 

```
{
  "address": "0x742d35Cc6634C0532925a3b844Bc9e7595f0bEb",
  "privateKey": "your-private-key-here-do-not-share",
  "balance": 1000.5,
  "totalEarned": 250.75,
  "totalSpent": 45.25,
  "pendingTransactions": 2,
  "exportDate": "2025-09-10T18:36:14.742Z",
  "version": "1.0"
}
```

Images displaying the success and error messages: 
<img width="1226" height="325" alt="image" src="https://github.com/user-attachments/assets/bc460efe-02f4-4405-a86b-a5915634ad39" />

<img width="1235" height="363" alt="image" src="https://github.com/user-attachments/assets/f6c15195-e81b-4615-b00f-62c63da2dace" />

turtle-jake-mallen commented:
Your branch is currently 90 commits behind `chiral-network/chiral-network:main`. You may want to rebase on top of `main`.

potato-khannah-shaltiel commented:
yup, sorry about that! i saw just after i pushed. i should have fixed it now

turtle-jake-mallen commented:
Maybe consider adding it here instead?
<img width="1178" height="950" alt="image" src="https://github.com/user-attachments/assets/c0f3f73e-18c8-4060-a43a-0548f120419b" />

The section with the private key at the bottom has since been moved to the top left.

potato-khannah-shaltiel commented:
removed it from bottom and added it to the chiral network wallet card 
<img width="1262" height="723" alt="image" src="https://github.com/user-attachments/assets/476f1493-eb3e-4459-84c6-57c1438bba72" />
<img width="1507" height="750" alt="image" src="https://github.com/user-attachments/assets/ae0b613c-7bbb-4e09-8237-ec3aae150c9a" />

pushing changes soon
